mav_msgs/include/mav_msgs/common.h

```diff
@@ -158,11 +158,50 @@ inline Eigen::Vector3d omegaFromRotationVector(
                 -rot_vec(1), rot_vec(0), 0.0f;
   // Set up matrix to calculate omega
   Eigen::Matrix3d W;
-  W = Eigen::MatrixXd::Identity(3,3) - phi_matrix * (1-cos(phi))/phi/phi
+  W = Eigen::MatrixXd::Identity(3,3) + phi_matrix * (1-cos(phi))/phi/phi
```

Comment:
Maybe add the reference also here, so people know where this formula comes from.